### ISSUE TITLE
JAMFPolicyEventID Clarification

### ISSUE THREAD

**Issue by dwshore** [IBM/mac-ibm-enrollment-app#7]
In the JAMFConstants.swift file, line 29:

> struct JAMFPolicyEventID {
>     static let softwareInstall = ""
>     static let removeFramework = ""
> }

It's not clear what to put in for either of these variables. Is there an expectation that there's one policy that installs everything? Or how do we define a different ID for each of the different bundles we want to deploy?

Thanks!

**Comment by jalatman** [comment]
You are correct. The customer is creating a list of selections. The policy event you are calling referenced from softwareInstall would look at that list and perform accordingly. In our environment we use event triggers rather than policy id numbers as that makes things more portable when moving between environments (qa to prod for example).
